app/src/main/java/vn/edu/usth/minigh/RepoActivity.kt

```diff
@@ -0,0 +1,60 @@
+package vn.edu.usth.minigh
+
+import android.os.Bundle
+import android.content.Intent
+import android.view.View
+import android.widget.TextView
+
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import androidx.viewpager2.widget.ViewPager2
+
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
+
+import vn.edu.usth.minigh.BaseActivity
+import vn.edu.usth.minigh.fragments.IssuesListFragment
+import vn.edu.usth.minigh.fragments.PRsListFragment
+import vn.edu.usth.minigh.fragments.RepoLogFragment
+import vn.edu.usth.minigh.fragments.RepoSummaryFragment
+import vn.edu.usth.minigh.fragments.RepoTreeFragment
+
+val TAB_NAMES = arrayOf("Summary", "Tree", "Log", "Issues", "PRs")
```

Comment:
Since we're making a client for GitHub, wouldn't it make more sense if we use their terms (i.e. README, Source code, Commits) instead?

Reply:
Nvm in Graph they also call it tree

Reply:
I'm going to pretend that I haven't seen this :smile:  Jokes aside GitHub does call it tree (see the URL) and for the log, *commits* is a bit to vague: *history* might be a better non-technical terms but :shrug: